requirements.txt

```diff
@@ -9,32 +9,33 @@ constantly==15.1.0
 cracklib==2.9.3
 cryptography==2.7
 dnspython==1.16.0
+emoji==0.5.3
 hyperlink==19.0.0
 idna==2.8
 incremental==17.5.0
 Jinja2==2.10.1
-ldap3==2.6
+ldap3==2.6.1
 MarkupSafe==1.1.1
-ocflib==2019.6.25.11.44
+ocflib==2019.9.1.11.30
```

Comment:
Huh, I thought `ocflib` was excluded from this list because we want to just use the latest, but maybe that broke somewhere or wasn't set up in this repo properly? Anyway, not really related to this review, just strange and wanted to mention it

Reply:
Ok, this is just done [in ocfweb](https://github.com/ocf/ocfweb/blob/9670d1d408b1292c85f7ea263ab11fefc75ac8ba/Makefile#L92) but hasn't been done here properly. Looks like I did have this set up correctly in the [initial commit](https://github.com/ocf/slackbridge/commit/d1edc25e4d1340ecd6de494d5ef79ddc35806085#diff-b67911656ef5d18c4ae36cb6741b7965R21) but then messed it up when ocflib was removed and [I readded it](https://github.com/ocf/slackbridge/commit/939066d111623eb94ca382d8f6aaccdc5a63011d#diff-b4ef698db8ca845e5845c4618278f29aR18) earlier this summer.

Reply:
Yeah just leave this as-is (changed to a newer version) and then a separate review should fix this issue